projects/angular/src/forms/datepicker/date-input.spec.ts

```diff
@@ -120,7 +120,7 @@ export default function () {
         it('should refocus input after selecting a date for a11y', async () => {
           const input: HTMLInputElement = context.testElement.querySelector('input');
           expect(document.activeElement).not.toBe(input);
-          dateNavigationService.notifySelectedDayChanged(new DayModel(2019, 1, 1));
+          dateNavigationService.notifySelectedDayChanged(new DayModel(2019, 1, 1), true);
```

Comment:
```suggestion
          dateNavigationService.notifySelectedDayChanged(new DayModel(2019, 1, 1));
```

We do not need to pass `true` everywhere because the `emitEvent` parameter has a default value. Feel free to cherry-pick 0e9ab2cfad8d19507b5ef64f3e150454f335b6dd.